.github/workflows/it-test.yaml

```diff
@@ -30,7 +30,7 @@ on:
       php-branch:
         required: false
         type: string
-        default: "staging"
+        default: "main"
```

Comment:
Do we need to update the secrets as well?
```
          aws-account-id: ${{ secrets.STAGING_AWS_ACCOUNT_ID }}
````

Reply:
Sharing here for transparency

https://monta-app01.slack.com/archives/C04HQ96GZ8W/p1728387931346109?thread_ts=1728386969.313129&cid=C04HQ96GZ8W